openedx/core/djangoapps/user_authn/api/tests/test_serializers.py

```diff
@@ -135,3 +204,35 @@ def test_mfe_context_serializer(self):
             output_data,
             expected_data
         )
+
+    def test_get_mfe_context_response_keys(self):
```

Comment:
Should we not move this test to `test_views.py` file, because this file covers only sterilizers tests.
Please make sure this test is not already written there.

Reply:
I added this in the serializer's test because the keys are modified after the response is serialized.